apps/src/studio/Item.js

```diff
@@ -315,10 +340,14 @@ Item.prototype.removeElement = function() {
 };
 
 /**
- * Stop our animations
+ * Retrieve animation frame duration (frames per tick)
  */
-Item.prototype.stopAnimations = function() {
-  this.animation_.stopAnimator();
+Item.prototype.getAnimationFrameDuration = function () {
+  if (this.dir === Direction.NONE) {
+    return this.normalFrameDuration;
```

Comment:
Yes, `speed` has always meant `movementSpeed`. We certainly could change that, but will need to do that across a lot of code (`Collidable`, `Item`, `Projectile`, and now `Sprite`).

The game-level logic now adds the concept of changing animation rate based on whether or not the `Item` or `Sprite` is moving. We could also modify the `StudioAnimation` class to support a different animation rate for each animation, but that still doesn't allow us to dynamically slow them down or speed them up as we alter movement speed.